docs/100_producten/100_borgstelling/borgstelling-beeindigen.feature

```diff
@@ -1,14 +1,27 @@
 # language: nl
 Functionaliteit: Borgstelling beëindigen
 
-  Achtergrond: 
-    Gegeven het kenmerk "093e08db-6791-454b-a172-068099514907"
+  Achtergrond:
+    Gegeven een kredietbank
 
-  Scenario: Borgstelling beëindigen
+  Scenario: Borgstelling beëindigen van afgegeven aanvraag
     Gegeven een afgegeven borgstelling
+    En een beeindigdatum van "2022-01-01"
     Wanneer het "beëindiging borgstelling" bericht is verstuurd via het Schuldenknooppunt
```

Comment:
```suggestion
    Wanneer het "beëindigen borgstelling" bericht is verstuurd via het Schuldenknooppunt
```

Reply:
Overal wordt gerept over "beëindiging borgstelling" en niet beëindigen. Zie ook schuldenknooppunt portaal. Wil je dit evengoed aangepast hebben?